docs/guides/runtime-options-overview.mdx

```diff
@@ -18,288 +19,298 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
+| [resilience](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptions)           |                          |      |                               |                      | resilience    |
+|                      | [layer_noise_learning](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.LayerNoiseLearningOptions)     |      | `True`/`False`                | `True`               |      |
+|                      |                          | max_layers_to_learn     |  `None`/ integer >= 1                                                                      | `4`                    |                      |
+|                      |                          | shots_per_randomization |  integer >= 1                                                                         |  `128`                    |                      |
+|                      |                          | num_randomizations      |   integer >= 1                                                                            |  `32`                    |                      |
+|                      |                          | layer_pair_depths       |  list[int] of 2-10 values in the range [0, 200]                           | `(0, 1, 2, 4, 16, 32)`              |                      |
+|                      | measure_mitigation       |        | `True`/`False`                      | `True`                      |                      |
+|                      | [measure_noise_learning](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.MeasureNoiseLearningOptions)   | num_randomizations      |  integer >= 1  | `32`                      |                      |
+|                      |                          | shots_per_randomization |  integer                                                                                | 'auto'                    |                      |
+|                      | [pec](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.PecOptions)   | max_overhead            |  `None`/ integer >1          | `100`                      |                      |
+|                      |                          | noise_gain              |  `auto`/ float in the range `[0, 1]`                                                     | `auto`                    |                      |
+|                      | pec_mitigation           |                         | `True`/`False`                                                                        | `False`                      |                      |
+|                      | zne_mitigation           |                         | `True`/`False`                                                                         | `False`                     |                      |
+|                      | [zne](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ZneOptions)                      | noise_factors           |   list of floats; each float >= 1     | `(1, 3, 5)`           |                      |
```

Comment:
nit... `pec_mitigation` should go before `pec`, like how `zne_mitigation` is before `zne`.